src/services/httpsRequestUtils.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { base64Encode } from './httpsRequestUtils'
+
+describe('#base64Encode', () => {
```

Comment:
I'm not sure what the lambda expects, but I think making the request with at least one valid and one invalid request would make for interesting test cases. Suggestions:
> Missing headers, empty body?